packages/widget-chat/src/components/activity-title-bar/index.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {findRenderedDOMComponentWithClass, renderIntoDocument} from 'react-addons-test-utils';
+
+import ActivityTitleBar from '.';
+
+let user;
+let component;
+
+describe(`ActivityTitleBar component`, () => {
```

Comment:
hmm, your first sentence disagrees with itself :)

each message is single activity, so I'd assume ActivityTitleBar would be a title bar for a given message.  ActivityListTitleBar or ConversationTitleBar seems like a more appropriate name for the bar that appears at the top of a list of messages.